src/main/kotlin/no/nav/familie/ef/sak/integration/FamilieIntegrasjonerClient.kt

```diff
@@ -45,6 +47,11 @@ class FamilieIntegrasjonerClient(@Qualifier("azure") restOperations: RestOperati
                                                           personIdentHeader(ident)).data!!
     }
 
+    fun egenAnsatt(ident: String): Boolean {
+        return postForEntity<Ressurs<EgenAnsattResponse>>(integrasjonerConfig.egenAnsattUri,
+                                                          EgenAnsattRequest(ident)).data!!.erEgenAnsatt
+    }
```

Comment:
Spør når jeg ser en !! - er vi sikre på at vi skal bruke denne her ;)

Reply:
Løses i en annen pull request for å løse dette på flere steder

Reply:
ikke etter at vi utvidet Ressurs med metoden .getDataOrThrow()

Reply:
@bent-lorentzen er ikke helt sikker på at det blir helt riktig å kaste en "IllegalStateException" fra den, lurer på om vi burde endre om den til å kaste en kontrollert exception som vi kan ha en exceptionmapper for i stedet og returnere en bedre feilmelding til frontend

Reply:
Det går jo selvfølgelig.